src/main/java/edu/kis/powp/jobs2d/command/CommandParser.java

```diff
@@ -0,0 +1,9 @@
+package edu.kis.powp.jobs2d.command;
+
+import java.io.FileNotFoundException;
+import java.util.List;
+
+public interface CommandParser {
+	void fillListFromFile() throws FileNotFoundException;
```

Comment:
I do not see a reason to couple parser with a file system.

Reply:
changed